roles/hostapd/templates/hostapd.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Hostapd {{ item }}       IEEE 802.11 AP, IEEE 802.1X/WPA/WPA2/EAP/RADIUS Authenticator
+Conflicts=hostapd.service {% if item == "open" %}hostapd_secure{% else %}hostapd_open{% endif %}.service
+After=network.target hostapd.service {% if item == "open" %}hostapd_secure.service{% endif %}
```

Comment:
I'm not sure assignment based on test is correct (item=open, assigne hostapd_secure)

Reply:
Yes, it supposed to stop the hostapd.service and hostapd_secure.service (avoid conflict).
In this role, you have three services:
- hostapd.service
- hostapd_open.service
- hostapd_secure.service
When one is started, the two others need to be stopped.

The main idea was to use `service hostapd_secure start` to stop `hostapd.service` and `hostapd_open.service` if one of them is running. Same for `service hostapd_open start` to stop `hostapd.service` and `hostapd_secure.service` if one of them is running.
Another way is to stop, unwanted services and start the wanted service.

Reply:
There is a circle around this part.
If you start on, you need to stop the other, but at one point, you need to exit from that circle. The `After=` with only `hostapd_secure.service` on `hostapd_open.service`.